app/Models/ReceivedRepayment.php

```diff
@@ -23,7 +23,13 @@ class ReceivedRepayment extends Model
      * @var array
      */
     protected $fillable = [
-        //
+        'amount',
+        'currency_code',
+        'received_at',
```

Comment:
there might have been a reason for empty `fillable`, although it's not clear from the code here

Reply:
yes, it can also be the constraint that we need to comply with in insert or update statements called from eloquent